server/lib/scene/scene.actions.js

```diff
@@ -189,6 +189,11 @@ const actionsFunc = {
     const textWithVariables = Handlebars.compile(action.text)(scope);
     await self.message.sendToUser(action.user, textWithVariables);
   },
+  [ACTIONS.MESSAGE.SEND_CAMERA]: async (self, action, scope) => {
+    const textWithVariables = Handlebars.compile(action.text)(scope);
+    const image = await self.device.camera.getImage(action.camera);
```

Comment:
The problem with this function is that it's taking a cached camera image, not the live camera image.

In scene, you probably want to do actions like "If there is a motion in the kitchen => Send camera image of the kitchen" 

The camera image should be the live camera image and not the image of 1 minute ago ! 

I think you need to write a new camera function like `camera.getLiveImage()` that get the live image

Reply:
Ho crap, I don't see what the function getImage return an 'old' image. I have made the change now :)